### ISSUE TITLE
Sign-up Errors

### ISSUE THREAD

**Issue by sebrittain** [sebrittain/referral-love#1]
You cannot setup a new account on the website.  I think this has to do with a database error I get when I run rake db:migrate.  I get the following :  
Migrating to AddConfirmationToUsers 
add_column(:users, :password, :password_confirmation) 
rake aborted!
PG::Error:ERROR: type "password_confirmatin" does not exist
LINE 1: ALTER TABLE "users" ADD COLUMN"password" password_confirmation
:

**Comment by aaronroe** [comment]
What happened to the colors of the site?  i'll take a look at it.

**Comment by aaronroe** [comment]
why does my page look so different than the one on heroku, even though i pulled and it says i am up to date?

**Comment by sebrittain** [comment]
My fault, I pushed it to heroku, but not to git.  I'm trying to improve the CSS for the site.  I managed to get rid of the weird blank space and make the colors work better.  If you want to switch anything around let me know.

**Comment by aaronroe** [comment]
that's strange... I just did pull to latest and i tried to signup.  it worked?  try pushing to heroku again and see if anything changes..

**Comment by sebrittain** [comment]
Yeah, it works on the local host, but if you try to do it on heroku it fails.

**Comment by aaronroe** [comment]
that is quite strange.  was it ever working on heroku?
